onesignal-admin.php

```diff
@@ -272,12 +272,12 @@ public static function onesignal_notif_on_post_html_view($post)
         $settings_send_notification_on_wp_editor_post = $onesignal_wp_settings['notification_on_post'];
 
         /* This is a scheduled post and the user checked "Send a notification on post publish/update". */
-        $post_metadata_was_send_notification_checked = (get_post_meta($post->ID, 'onesignal_send_notification') === true);
+        $post_metadata_was_send_notification_checked = (get_post_meta($post->ID, 'onesignal_send_notification', true) === '1');
 
         // We check the checkbox if: setting is enabled on Config page, post type is ONLY "post", and the post has not been published (new posts are status "auto-draft")
         $meta_box_checkbox_send_notification = ($settings_send_notification_on_wp_editor_post &&  // If setting is enabled
```

Comment:
Could we break up statement up a bit since we are currently revisiting it in this PR?
One suggestion is we can start with `if (!$post_metadata_was_send_notification_checked)` as we don't need to bother if it was already checked. `$post_metadata_was_send_notification_checked` is currently in this statement as an OR but is at the end which is confusing.